### PR TITLE
Fixed the wrong directory structure and use static version of netty-tcnative

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

**Comment by tmatsuo** [comment]
@hsaliak

Can you take a look? Do we need to use `-boringssl-static`?

File: grpc/pom.xml [hunk 1]
```diff
@@ -127,8 +127,16 @@
       <version>1.21.0</version>
     </dependency>
     <dependency>
+      <!--
+        It is recommended to use OpenSSL: Statically Linked Netty
+        for transport security.  These steps do not use the TLS that
+        comes with JDK (Jetty APLN/NPN), which is not
+        recommended. See
+        https://github.com/grpc/grpc-java/blob/v0.13.x/SECURITY.md
```
Comment (eschapira) [inline grpc/pom.xml]: Change this path to https://github.com/grpc/grpc-java/blob/master/SECURITY.md

**Comment by eschapira** [comment]
LGTM. Make sure to update the path for the security docs.

**Comment by tmatsuo** [comment]
Thanks I'll merge this once the test passes

**Pull request merged**